packages/core-mobile/app/hooks/earn/useImportAnyStuckFunds.ts

```diff
@@ -20,21 +23,33 @@ export const useImportAnyStuckFunds = (
   const activeAccount = useSelector(selectActiveAccount)
   const isDevMode = useSelector(selectIsDeveloperMode)
   const selectedCurrency = useSelector(selectSelectedCurrency)
+  const activeNetwork = useSelector(selectActiveNetwork)
+  const { defaultFeeState } = useGetFeeState()
+  const devnet = isDevnet(activeNetwork)
 
   return useQuery({
     // no need to retry failed request as we are already doing interval fetching
     retry: false,
     enabled,
     refetchInterval: REFETCH_INTERVAL,
     // eslint-disable-next-line @tanstack/query/exhaustive-deps
-    queryKey: ['ImportAnyStuckFunds', activeAccount, isDevMode],
+    queryKey: [
+      'ImportAnyStuckFunds',
+      activeAccount,
+      isDevMode,
+      activeNetwork,
+      defaultFeeState,
```

Comment:
let's not use active network since it is an object. also I think you can just remove activenetwork and default fee state from the array. we don't want this logic to rerun for those 2 anyway.

Reply:
removed activeNework,  don't we still need to rerun when we get the defaultFeeState?